src/main/java/calculator/model/delimiter/Delimiters.java

```diff
@@ -0,0 +1,43 @@
+package calculator.model.delimiter;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+import calculator.config.DefaultDelimiter;
+import calculator.validation.InputValidator;
+
+public class Delimiters {
+	private final Set<String> delimiters = new HashSet<>();
+
+	public Delimiters() {
+		initDefaultDelimiters();
+	}
+
+	public void addDelimiter(String delimiterInput) {
+		InputValidator.validateEmpty(delimiterInput);
+		InputValidator.validateNonDigit(delimiterInput);
+		delimiters.add(delimiterInput);
+	}
+
+	public boolean isRegisteredDelimiter(String delimiter) {
+		return delimiters.contains(delimiter);
+	}
+
+	public String[] getRegex() {
+		return delimiters.stream()
+			.map(Pattern::quote)
+			.toArray(String[]::new);
+	}
```

Comment:
```suggestion
	public String[] getRegexes() {
		return delimiters.stream()
			.map(Pattern::quote)
			.toArray(String[]::new);
	}
```

배열이나 컬렉션을 반환하는 메서드는 복수형으로 작성하면 어떨까요?

Reply:
오! 말씀대로 복수형이 더 직관적일 것 같습니다. 놓친 부분 알려주셔서 감사합니다!